sdk/ai/ai-projects/package.json

```
@@ -69,30 +69,24 @@
     "@azure/core-tracing": "^1.2.0",
     "@azure/core-util": "^1.9.0",
     "@azure/logger": "^1.1.4",
-    "@azure/opentelemetry-instrumentation-azure-sdk": "^1.0.0-beta.7",
     "@azure/storage-blob": "^12.26.0",
     "@azure-rest/ai-inference": "^1.0.0-beta.6",
     "@azure-rest/core-client": "^2.1.0",
-    "@opentelemetry/api": "^1.9.0",
-    "@opentelemetry/api-logs": "^0.200.0",
-    "@opentelemetry/exporter-logs-otlp-grpc": "^0.200.0",
-    "@opentelemetry/exporter-trace-otlp-grpc": "^0.200.0",
-    "@opentelemetry/instrumentation": "^0.57.0",
-    "@opentelemetry/sdk-logs": "^0.200.0",
-    "@opentelemetry/sdk-trace-node": "^1.30.0",
-    "@traceloop/instrumentation-langchain": "^0.13.0",
-    "@traceloop/instrumentation-openai": "^0.13.0",
     "openai": "^4.84.1",
     "tslib": "^2.6.2"
   },
   "devDependencies": {
     "@azure/dev-tool": "^1.0.0",
     "@azure/eslint-plugin-azure-sdk": "^3.0.0",
     "@azure/identity": "^4.3.0",
-    "@azure/monitor-opentelemetry-exporter": "^1.0.0-beta.32",
+    "@azure/monitor-opentelemetry": "^1.11.1",
+    "@azure/opentelemetry-instrumentation-azure-sdk": "^1.0.0-beta.7",
```

we shouldn't depend on floating beta versions:
```suggestion
    "@azure/opentelemetry-instrumentation-azure-sdk": "1.0.0-beta.7",
```